tests/emukit/test_acquisitions.py

```diff
@@ -9,6 +9,9 @@
 
 import numpy as np
 import pytest
+
+pytest.importorskip("GPy")
```

Comment:
I am not sure how i feel about this skip. If a developer did not install GPy, this will hide too many potential errors.